src/components/Reports/StaffExpenseReport/Helpers/StaffReportEnum.ts

```diff
@@ -0,0 +1,22 @@
+export enum TableType {
+  Income = 'income',
+  Expenses = 'expenses',
+}
+
+export enum CategoryType {
+  Benefit = 'benefit',
+  Contribution = 'contribution',
+  Salary = 'salary',
+}
+
+export enum DateRange {
+  WeekToDate = 'Week to Date',
+  MonthToDate = 'Month to Date',
+  YearToDate = 'Year to Date',
```

Comment:
Enum values should be formatted for computers not for humans. If we want a human-readable value, we should be doing that explicitly and localizing the labels.

```suggestion
  WeekToDate = 'weekToDate',
  MonthToDate = 'monthToDate',
  YearToDate = 'yearToDate',
```